glusterd2/commands/volumes/volume-create-txn.go

```diff
@@ -122,6 +122,7 @@ func newVolinfo(req *api.VolCreateReq) (*volume.Volinfo, error) {
 		ID:        uuid.NewRandom(),
 		Name:      req.Name,
 		VolfileID: req.Name,
+		Capacity:  req.Size,
```

Comment:
This only works for auto provisioned volumes. Are you planning to support for manually provisioned volume as well?

Reply:
thought of handling it in a separate PR

Reply:
That's not a priority now. Just saying.

Reply:
Even though that's not a priority, response structure for a non-smart volume would report back with capacity as 0? Is there a way not to return capacity field in case of non smart volume till that's implemented?

Reply:
I have stopped it from being used if the value is zero which is the case for manually provisioned volume

Reply:
> Is there a way not to return capacity field in case of non smart volume till that's implemented?

Yes there is a way and [has been suggested](https://github.com/gluster/glusterd2/pull/1193#issuecomment-419397649).